### PR TITLE
Certified accuracy #699

### LINKED ISSUE #699: Certified accuracy
In the computation of certified accuracy located in [https://github.com/Trusted-AI/adversarial-robustness-toolbox/blob/ddfb4e29329891731ff2f7d36b114be784db0d03/notebooks/output_randomized_smoothing_mnist.ipynb]
the samples with the radius zero are removed:
    `num_cert = len(np.where(radius > 0)[0])`

So in the testset if there is only one sample with positive certified radius, the certified accuracy would be `100%` in the interval [0, r], where r is the certified radius of that specific sample. In a nutshell, I believe the samples with zero radii should not be removed from the computation of certified accuracy.

### PR DESCRIPTION
**Pull request by ehsankf**
# Description

Please include a summary of the change, motivation and which issue is fixed. Any dependencies changes should also be included.

Fixes #699

## Type of change

Please check all relevant options.

- [x] Improvement (non-breaking)

# Testing

Please describe the tests that you ran to verify your changes. Consider listing any relevant details of your test configuration.

- [ ] Test A
- [ ] Test B

**Test Configuration**:
- OS
- Python version
- ART version or commit number
- TensorFlow / Keras / PyTorch / MXNet version

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by beat-buesser** [comment]
Hi @ehsankf Thank you very much for this pull request!

**Review by beat-buesser — APPROVED** [review]
